ansible/roles/ocp4-workload-acm-app-policies-lab/files/wait-for-deployment

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+READY=0
+WAIT=0
+MAX_WAIT=300
+CLUSTER="$1"
+DEPLOYMENT_NAMESPACE="$2"
+DEPLOYMENT_NAME="$3"
+echo "Checking if deployment ${DEPLOYMENT_NAME} from namespace ${DEPLOYMENT_NAMESPACE} on cluster ${CLUSTER} is ready"
```

Comment:
Can we use k8s means for this instead of shell scripts? Logging workload has examples how to.

Reply:
I asked the same question on chat (agnosticd room)>
Those scripts are for students during class, not automation

Reply:
Oh. That wasn’t obvious. :-)